composer.json

```diff
@@ -299,7 +299,27 @@
         },
         "drupal": {
             "type": "composer",
-            "url": "https://packages.drupal.org/8"
+            "url": "https://packages.drupal.org/8",
+            "exclude": [
+                "drupal/acquia_cms_article",
```

Comment:
@rajeshreeputra i dont think we need to add this exclude section , because composer install will automatically take all the changes from composer.lock and hence updating composer.lock is enough.
cc @vishalkhode1